mne/viz/_3d.py

```diff
@@ -432,14 +435,15 @@ def plot_trans(info, trans='auto', subject=None, subjects_dir=None,
                         continue
                 logger.info('Using %s for head surface.' % this_surf)
                 rr, tris = read_surface(surf_fname)
-                # Normalization ensured later.
-                head_surf = {'rr': rr / 1000., 'tris': tris, 'nn': rr.copy(),
-                             'coord_frame': FIFF.FIFFV_COORD_MRI}
+                head_surf = dict(rr=rr / 1000., tris=tris, ntri=len(tris),
```

Comment:
... and instead of doing this, you can use `read_surface(..., return_dict=True)`